catalyst_voices/packages/catalyst_voices_shared/lib/src/platform_widget_builder/platform_widget_builder.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:catalyst_voices_shared/src/platform/catalyst_platform.dart';
+import 'package:flutter/widgets.dart';
+
+class PlatformWidgetBuilder<T> extends StatelessWidget {
+  final Widget Function(BuildContext context, T? data) builder;
+  final T? android;
```

Comment:
I am just wondering, do we really need all these fields ?
Because during the widget `build` we are totally relying on the `CatalystPlatform`.